serhii.trokhymets/Home_work 3/task_1.rb

```diff
@@ -0,0 +1,36 @@
+class Easy_number
```

Comment:
Please, use [CamelCase for classes naming](https://github.com/rubocop-hq/ruby-style-guide#camelcase-classes)